custom_components/adjustable_bed/coordinator.py

```
@@ -738,7 +745,7 @@ def _on_disconnect(self, client: BleakClient) -> None:
         # Capture controller reference before clearing to avoid race condition
         controller = self._controller
         if controller is not None and hasattr(controller, 'stop_keepalive'):
-            asyncio.create_task(controller.stop_keepalive())
+            asyncio.create_task(controller.stop_keepalive())  # type: ignore[attr-defined]
```

Ruff (0.14.13)

Store a reference to the return value of `asyncio.create_task`

(RUF006)